plugins/cookie_manager/lib/src/cookie_mgr.dart

```diff
@@ -28,7 +28,25 @@ class CookieManager extends Interceptor {
   final CookieJar cookieJar;
 
   /// Merge cookies into a Cookie string.
+  /// Cookies with longer paths are listed before cookies with shorter paths.
   static String getCookies(List<Cookie> cookies) {
+    // Remove previous cookie with same name.
+    cookies = cookies.fold<List<Cookie>>([], (prev, current) {
+      // Check if the current cookie's name is already in the list.
+      if (prev.any((c) => c.name == current.name)) {
+        // Skip this cookie.
+        return prev;
+      } else {
+        // Add this cookie to the list.
+        return [...prev, current];
+      }
+    });
+
```

Comment:
nit: We don't need so many blank lines for inline codes, only for blocks like `test(...)`. Also consider to apply the suggestion to other codes.

Reply:
noted.